src/storage/txn/scheduler.rs

```diff
@@ -374,6 +392,9 @@ impl<E: Engine, L: LockManager, P: PdClient + 'static> Scheduler<E, L, P> {
                             "process cmd with snapshot";
                             "cid" => task.cid, "cb_ctx" => ?cb_ctx
                         );
+                        SCHED_PRE_HANDLE_1_DURATIONS_VEC
```

Comment:
Could it have a more descriptive name instead of 1, 2, 3... For example `SCHED_AFTER_SNAPSHOT_DURATIONS_VEC` because this is right after getting snapshot.

Reply:
Addressed